src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  errorType: ErrorType;
+};
+
+export const Notifications: React.FC<Props> = ({ errorType }) => (
```

Comment:
```suggestion
export const Notifications: FC<Props> = ({ errorType }) => (
```
import { FC } from react

Reply:
So we have a choice to use React.FC just once, or import React. I'll leave it as it is for now because in this case it doesn't require another row of code like import does.